2025-02-28 21:02:47.264 [main] INFO  ad.hibernate.hmarort.Main - Iniciando aplicación de gestión de pedidos
2025-02-28 21:02:47.268 [main] INFO  ad.hibernate.hmarort.Main - Interfaz seleccionada: automatica
2025-02-28 21:02:47.290 [main] INFO  com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Starting...
2025-02-28 21:02:47.430 [main] INFO  com.zaxxer.hikari.pool.HikariPool - HikariPool-1 - Added connection org.sqlite.jdbc4.JDBC4Connection@176b75f7
2025-02-28 21:02:47.432 [main] INFO  com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Start completed.
2025-02-28 21:02:47.432 [main] INFO  a.h.h.database_config.SQLiteConfig - Iniciando inicialización de base de datos SQLite
2025-02-28 21:02:47.451 [main] WARN  a.h.h.database_config.SQLiteConfig - Advertencia al ejecutar statement: [SQLITE_ERROR] SQL error or missing database (table Zonas_Envio already exists)
2025-02-28 21:02:47.452 [main] WARN  a.h.h.database_config.SQLiteConfig - Advertencia al ejecutar statement: [SQLITE_CONSTRAINT_PRIMARYKEY] A PRIMARY KEY constraint failed (UNIQUE constraint failed: Zonas_Envio.id_zona)
2025-02-28 21:02:47.452 [main] WARN  a.h.h.database_config.SQLiteConfig - Advertencia al ejecutar statement: [SQLITE_CONSTRAINT_PRIMARYKEY] A PRIMARY KEY constraint failed (UNIQUE constraint failed: Clientes.id_cliente)
2025-02-28 21:02:47.452 [main] INFO  a.h.h.database_config.SQLiteConfig - Base de datos inicializada correctamente
2025-02-28 21:02:47.456 [main] INFO  a.h.hmarort.utils.SessionManager - Inicializando SessionFactory de Hibernate
2025-02-28 21:02:47.530 [main] INFO  org.hibernate.Version - HHH000412: Hibernate ORM core version 6.4.1.Final
2025-02-28 21:02:47.739 [main] INFO  o.h.c.i.RegionFactoryInitiator - HHH000026: Second-level cache disabled
2025-02-28 21:02:47.818 [main] WARN  o.hibernate.orm.connections.pooling - HHH10001002: Using built-in connection pool (not intended for production use)
2025-02-28 21:02:47.819 [main] INFO  o.hibernate.orm.connections.pooling - HHH10001005: Loaded JDBC driver class: org.sqlite.JDBC
2025-02-28 21:02:47.819 [main] INFO  o.hibernate.orm.connections.pooling - HHH10001012: Connecting with JDBC URL [jdbc:sqlite:src/main/resources/pedidos.db]
2025-02-28 21:02:47.819 [main] INFO  o.hibernate.orm.connections.pooling - HHH10001001: Connection properties: {}
2025-02-28 21:02:47.819 [main] INFO  o.hibernate.orm.connections.pooling - HHH10001003: Autocommit mode: false
2025-02-28 21:02:47.822 [main] INFO  o.hibernate.orm.connections.pooling - HHH10001115: Connection pool size: 20 (min=1)
2025-02-28 21:02:48.796 [main] INFO  o.hibernate.orm.connections.access - HHH10001501: Connection obtained from JdbcConnectionAccess [org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator$ConnectionProviderJdbcConnectionAccess@5763a655] for (non-JTA) DDL execution was not in auto-commit mode; the Connection 'local transaction' will be committed and the Connection will be set into auto-commit mode.
2025-02-28 21:02:48.807 [main] INFO  a.h.hmarort.utils.SessionManager - SessionFactory inicializada correctamente
2025-02-28 21:02:48.808 [main] INFO  a.h.hmarort.factory.DAOFactory - DAOFactory inicializado correctamente
2025-02-28 21:02:48.808 [main] INFO  ad.hibernate.hmarort.ui.UIAutoImpl - [GENERACIÓN AUTOMÁTICA DE DATOS]
2025-02-28 21:02:48.887 [main] INFO  a.h.h.d.i.DAOZonaEnvioImpl - Zona de envío insertada con ID: 9
2025-02-28 21:02:48.897 [main] INFO  a.h.h.d.i.DAOZonaEnvioImpl - Zona de envío insertada con ID: 10
2025-02-28 21:02:48.898 [main] INFO  a.h.h.d.i.DAOZonaEnvioImpl - Zona de envío insertada con ID: 11
2025-02-28 21:02:48.900 [main] INFO  ad.hibernate.hmarort.ui.UIAutoImpl - ✓ Zonas de envío generadas
2025-02-28 21:02:48.902 [main] INFO  a.h.h.d.i.DAOClienteImpl - Cliente insertado con ID: 15
2025-02-28 21:02:48.905 [main] INFO  a.h.h.d.i.DAOClienteImpl - Cliente insertado con ID: 16
2025-02-28 21:02:48.906 [main] INFO  a.h.h.d.i.DAOClienteImpl - Cliente insertado con ID: 17
2025-02-28 21:02:48.909 [main] INFO  a.h.h.d.i.DAOClienteImpl - Cliente insertado con ID: 18
2025-02-28 21:02:48.911 [main] INFO  a.h.h.d.i.DAOClienteImpl - Cliente insertado con ID: 19
2025-02-28 21:02:48.911 [main] INFO  ad.hibernate.hmarort.ui.UIAutoImpl - ✓ Clientes generados
2025-02-28 21:02:48.912 [main] INFO  a.h.h.d.implementacion.DAOPedidoImpl - Pedido insertado con ID: 18
2025-02-28 21:02:48.915 [main] INFO  a.h.h.d.implementacion.DAOPedidoImpl - Pedido insertado con ID: 19
2025-02-28 21:02:48.917 [main] INFO  a.h.h.d.implementacion.DAOPedidoImpl - Pedido insertado con ID: 20
2025-02-28 21:02:48.918 [main] INFO  a.h.h.d.implementacion.DAOPedidoImpl - Pedido insertado con ID: 21
2025-02-28 21:02:48.920 [main] INFO  a.h.h.d.implementacion.DAOPedidoImpl - Pedido insertado con ID: 22
2025-02-28 21:02:48.922 [main] INFO  a.h.h.d.implementacion.DAOPedidoImpl - Pedido insertado con ID: 23
2025-02-28 21:02:48.924 [main] INFO  a.h.h.d.implementacion.DAOPedidoImpl - Pedido insertado con ID: 24
2025-02-28 21:02:48.927 [main] INFO  a.h.h.d.implementacion.DAOPedidoImpl - Pedido insertado con ID: 25
2025-02-28 21:02:48.929 [main] INFO  a.h.h.d.implementacion.DAOPedidoImpl - Pedido insertado con ID: 26
2025-02-28 21:02:48.931 [main] INFO  a.h.h.d.implementacion.DAOPedidoImpl - Pedido insertado con ID: 27
2025-02-28 21:02:48.934 [main] INFO  a.h.h.d.implementacion.DAOPedidoImpl - Pedido insertado con ID: 28
2025-02-28 21:02:48.936 [main] INFO  a.h.h.d.implementacion.DAOPedidoImpl - Pedido insertado con ID: 29
2025-02-28 21:02:48.937 [main] INFO  ad.hibernate.hmarort.ui.UIAutoImpl - ✓ Pedidos generados
2025-02-28 21:02:48.937 [main] INFO  ad.hibernate.hmarort.ui.UIAutoImpl - [INFORME AUTOMÁTICO]
2025-02-28 21:02:49.235 [main] ERROR a.h.h.d.i.DAOClienteImpl - Error al calcular la facturación del cliente
org.hibernate.HibernateException: Calling method 'createQuery' is not valid without an active transaction (Current status: NOT_ACTIVE)
	at org.hibernate.context.internal.ThreadLocalSessionContext$TransactionProtectionWrapper.invoke(ThreadLocalSessionContext.java:339)
	at jdk.proxy2/jdk.proxy2.$Proxy54.createQuery(Unknown Source)
	at ad.hibernate.hmarort.dao.implementacion.DAOClienteImpl.calcularFacturacionTotalCliente(DAOClienteImpl.java:153)
	at ad.hibernate.hmarort.ui.UIAutoImpl.generarInformesAutomaticos(UIAutoImpl.java:115)
	at ad.hibernate.hmarort.ui.UIAutoImpl.iniciar(UIAutoImpl.java:61)
	at ad.hibernate.hmarort.Main.main(Main.java:42)
2025-02-28 21:02:49.237 [main] ERROR ad.hibernate.hmarort.ui.UIAutoImpl - Error en la ejecución automática
java.sql.SQLException: Error al calcular la facturación: Calling method 'createQuery' is not valid without an active transaction (Current status: NOT_ACTIVE)
	at ad.hibernate.hmarort.dao.implementacion.DAOClienteImpl.calcularFacturacionTotalCliente(DAOClienteImpl.java:160)
	at ad.hibernate.hmarort.ui.UIAutoImpl.generarInformesAutomaticos(UIAutoImpl.java:115)
	at ad.hibernate.hmarort.ui.UIAutoImpl.iniciar(UIAutoImpl.java:61)
	at ad.hibernate.hmarort.Main.main(Main.java:42)
Caused by: org.hibernate.HibernateException: Calling method 'createQuery' is not valid without an active transaction (Current status: NOT_ACTIVE)
	at org.hibernate.context.internal.ThreadLocalSessionContext$TransactionProtectionWrapper.invoke(ThreadLocalSessionContext.java:339)
	at jdk.proxy2/jdk.proxy2.$Proxy54.createQuery(Unknown Source)
	at ad.hibernate.hmarort.dao.implementacion.DAOClienteImpl.calcularFacturacionTotalCliente(DAOClienteImpl.java:153)
	... 3 common frames omitted
2025-02-28 21:02:49.237 [main] ERROR ad.hibernate.hmarort.ui.UIAutoImpl - Error en proceso automático: Error al calcular la facturación: Calling method 'createQuery' is not valid without an active transaction (Current status: NOT_ACTIVE)
2025-02-28 21:02:49.237 [main] INFO  ad.hibernate.hmarort.Main - Aplicación finalizada exitosamente
